python/lsst/ts/m2gui/model.py

```diff
@@ -55,19 +60,170 @@ def __init__(self, log):
         self.local_mode = LocalMode.Standby
         self.is_closed_loop = False
 
-        self.signal_error = SignalError()
+        self.signal_status = SignalStatus()
+
+        self.system_status = self._set_system_status()
+
+        self.fault_manager = FaultManager(self.get_actuator_default_status(False))
+
+    def _set_system_status(self):
+        """Set the default system status.
+
+        Returns
+        -------
+        system_status : `dict`
+            Default system status.
+        """
+
+        system_status = {
+            "isCrioConnected": False,
```

Comment:
I am not convinced this has to be a dictionary. Why those values cannot be member variables? You can use getattr/setattr to operate attributes by name, if that would be needed (to process JSON updates?)

Reply:
One of the reason is that the overview panel can add the indicators dynamically:
https://github.com/lsst-ts/ts_m2gui/blob/tickets/DM-34372/python/lsst/ts/m2gui/controltab/tab_overview.py#L80-L91

The other reason is that I do not like a class has too many attributes.